### PR TITLE
Refactor installation methods for fastlane

### PR DESCRIPTION
**Pull request by KrauseFx**
This is not done yet, but even if we automatically generate a Gemfile for the user, we still need to install fastlane for them.
1) to install fastlane and run `fastlane init`
2) to install `bundler` for them
See implementation https://github.com/fastlane/fastlane/pull/11582

So once it's installed, we'd then automatically setup and use a Gemfile for the user, and even with a proposal to automatically do `bundle exec` for the user with https://github.com/fastlane/fastlane/pull/11588

screen shot 2018-01-16 at 3 24 03 pm
screen shot 2018-01-16 at 3 23 51 pm
screen shot 2018-01-16 at 3 23 36 pm

Let me know what y'all think 👍

### CODE REVIEW

**Comment by taquitos** [comment]
❤️:goat:

**Comment by KrauseFx** [comment]
Not merging yet, this need to be applied to all other pages also, including the _fastlane_ code that generates the READMEs for match

File: docs/getting-started/ios/setup.md [hunk 1]
```diff
@@ -7,13 +7,17 @@ Make sure you have the latest version of the Xcode command line tools installed:
...
 ```
 
-### Choose your installation method:
+Install _fastlane_ using 
+```no-highlight
+[sudo] gem install fastlane -NV
+```
+or alternatively using `brew cask install fastlane`
+
+Navigate to your iOS or Android app and run
```
Comment (janpio) [inline docs/getting-started/ios/setup.md]: "iOS or Android app" => "project directory/folder" maybe?

**Comment by janpio** [comment]
Definitely like the simplicity.

(Note that this might get more complicated again with Windows and Linux support by the way - but we will tackle this when this is done. For now making it the best for macOS is the right thing to do.)

File: docs/getting-started/ios/setup.md (hunk 1 above)
Comment (KrauseFx) [reply docs/getting-started/ios/setup.md]: Nice, thanks

**Comment by KrauseFx** [comment]
Updating now

**Comment by KrauseFx** [comment]
Alright, this is ready now 👍

**Pull request merged**